tuna/machine.py

```diff
@@ -83,63 +93,73 @@ def __init__(self, **kwargs):
     self.__dict__.update(
         (key, value) for key, value in kwargs.items() if key in allowed_keys)
 
-    self.cnx_list = {}
-    self.log_list = {}
-    self.num_cpus = 0
+    self.cnx_list: dict = {}
+    self.log_list: dict = {}
+    self.num_cpus: int = 0
+    self.avail_gpus: List[Dict[int, str]]
+    self.sclk: int
+    self.mclk: int
+    self.gpus: List[Dict[Any, Any]]
+    self.cpus: List[dict]
```

Comment:
inconsistent with get_properties return type.

Reply:
corrected.

Reply:
Dict[Any, Any] = dict, please use most succinct notation.
The keys for these dicts are always str, so Dict[str, Any] would be the most specific.